### ISSUE TITLE
linchpin up breaks if Babel == 2.4.0

### ISSUE THREAD

**Issue by herlo** [CentOS-PaaS-SIG/linch-pin#214]
While working on testing, it appears osc-lib cannot have Babel 2.4.0. requirements.txt should be updated.

```
$ linchpin up
Traceback (most recent call last):
  File "/home/herlo/Projects/virtualenvs/refactor_tests/bin/linchpin", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/home/herlo/Projects/virtualenvs/refactor_tests/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3036, in <module>
    @_call_aside
  File "/home/herlo/Projects/virtualenvs/refactor_tests/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3020, in _call_aside
    f(*args, **kwargs)
  File "/home/herlo/Projects/virtualenvs/refactor_tests/lib/python2.7/site-packages/pkg_resources/__init__.py", line 3049, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/home/herlo/Projects/virtualenvs/refactor_tests/lib/python2.7/site-packages/pkg_resources/__init__.py", line 656, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/home/herlo/Projects/virtualenvs/refactor_tests/lib/python2.7/site-packages/pkg_resources/__init__.py", line 669, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/home/herlo/Projects/virtualenvs/refactor_tests/lib/python2.7/site-packages/pkg_resources/__init__.py", line 859, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (Babel 2.4.0 (/home/herlo/Projects/virtualenvs/refactor_tests/lib/python2.7/site-packages), Requirement.parse('Babel!=2.4.0,>=2.3.4'), set(['osc-lib']))
```

**Comment by greg-hellings** [comment]
Just literally encountered this less than 3 minutes ago. Pinning to 2.3.4
seems (so far) to work.

**Comment by herlo** [comment]
Fix is in #215. Requesting a slight change to that, but it should be fixed shortly.
